Core/src/main/java/com/paypal/android/core/CoreConfig.kt

```diff
@@ -2,6 +2,6 @@ package com.paypal.android.core
 
 data class CoreConfig(
     val clientId: String,
-    val clientSecret: String,
+    val clientSecret: String = "",
```

Comment:
Isn't clientSecret mandatory for merchants to add?

Reply:
well not in the case of `PayPalCheckout`, as we are doing server side integration only. Eventually we are going to need it.

Reply:
Client secret is actually not needed for client-side integration either. Since we're getting an access token through buyer authentication, we can call the create order API clientside.

Reply:
then if its not needed, should we remove `clientSecret` from `CoreConfig` as it is not an "universal" parameter? We can also just leave empty string as default value, or even null

Reply:
Yep, I think we can remove it completely. I believe it was originally added for the confirm payment source call before the API change was made to only require the client ID.

Reply:
Did we still want to remove `clientSecret` from `CoreConfig` in this PR?